commands/operator-sdk/cmd/new.go

```diff
@@ -235,3 +306,66 @@ func initGit() {
 	execCmd(os.Stdout, "git", "commit", "-q", "-m", "INITIAL COMMIT")
 	fmt.Fprintln(os.Stdout, "Run git init done")
 }
+
+// Copied from add/api.go command
+func updateRoleForResource(r *scaffold.Resource, absProjectPath string) error {
```

Comment:
If this is an exact copy of `updateRoleForResource`, I think this should live in `cmd/util` unless you have a reason to have another copy.

Reply:
It is an exact copy. I think that we can refactor in a bit.

Reply:
We can add a ToDo for now to consolidate them later.